app/src/components/widgets/charts/common/utils/chartRange.js

```diff
@@ -8,3 +8,13 @@ export const getLaunchAxisTicks = (itemsLength) => {
   }
   return array;
 };
+
+export const getTimelineAxisTicks = (itemsLength) => {
+  const start = itemsLength > 5 ? ((itemsLength / 5 / 2).toFixed() / 2).toFixed() : 0;
+  const step = itemsLength > 5 ? (itemsLength / 5).toFixed() : 1;
+  const result = [];
+  for (let iterator = start; iterator < itemsLength; iterator += step) {
```

Comment:
looks like function `range` from `common/utils/range`. Please also update `getLaunchAxisTicks` from this file to use `range`.